bin/generate-methods.js

```diff
@@ -15,9 +15,10 @@
 
```

Comment:
For the sake of maintainability you could move this the generator files to TS as well, your call tho.